subprojects/core/src/main/java/org/gradle/api/internal/file/copy/FileCopyDetailsInternal.java

```diff
@@ -22,4 +22,7 @@
 
     boolean isIncludeEmptyDirs();
 
+    default boolean isDefaultDuplicatesStrategy() {
+        throw new UnsupportedOperationException();
+    }
```

Comment:
There are only two implementations, and it's not something users should extend. I think the `default` implementation (especially one so punishing) is not warranted here.

Reply:
I removed the default method